code/game/objects/items/weapons/grenades/clusterbuster.dm

```diff
@@ -2,13 +2,17 @@
 //Clusterbang
 ////////////////////
 /obj/item/grenade/clusterbuster
-	desc = "Use of this weapon may constitute a war crime in your area, consult your local captain."
 	name = "clusterbang"
+	desc = "Use of this weapon may constitute a war crime in your area, consult your local captain."
 	icon = 'icons/obj/grenade.dmi'
 	icon_state = "clusterbang"
 	item_state = "flashbang"
 	var/payload = /obj/item/grenade/flashbang/cluster
 
+/obj/item/grenade/clusterbuster/examine(mob/user)
```

Comment:
This kinda feels like it might be better as an extended examine?

Reply:
It's important information and it isn't *too* long I feel. I take into account that many people won't attempt to perform an extended examine.